assets/js/blocks/price-filter/block.js

```diff
@@ -202,20 +202,38 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 			if ( prices[ 1 ] !== maxPrice ) {
 				setMaxPrice( prices[ 1 ] );
 			}
+
+			if (
+				filteringForPhpTemplate &&
+				hasSetPhpFilterDefaults &&
+				! attributes.showFilterButton
+			) {
+				onSubmit( prices[ 0 ], prices[ 1 ] );
```

Comment:
With this solution, moving the slider from `0` to `70` will cause 7 requests (`10`, `20`, `30`, ...), no? (At least, that's what I noticed in my testing)

Should we create a wrapper around `onSubmit` to debounce it?

Reply:
Thanks so much for this! It's my bad that I didn't test the block without the filter button. I address your review and this PR is ready to review again!